.travis.yml

```diff
@@ -42,7 +44,9 @@ matrix:
 
     - arch: arm64
       env:
-        - JOB="3.7, arm64" PYTEST_WORKERS=8 ENV_FILE="ci/deps/travis-37-arm64.yaml" PATTERN="(not slow and not network and not clipboard and not arm_slow)"
+        - JOB="3.7, arm64" ENV_FILE="ci/deps/travis-37-arm64.yaml" PATTERN="(not slow and not network and not clipboard and not arm_slow)"
```

Comment:
if this works great
but the worker config for arm is odd on travis

Reply:
Good to know. I'll play with it for a bit.

Reply:
Using `auto` would freeze the CI. I also tested this on a Raspberry Pi. Only the `test_fast.sh` script works on the Pi. Even though it's a 32bit arm system. But this could be universal for arm. Surprisingly, it happened on master today https://travis-ci.org/github/pandas-dev/pandas/jobs/729807957.